parsers/ES.py

```diff
@@ -177,17 +177,17 @@ def fetch_production(
                 # NOTE the LNG terminals are not built yet, so power generated by "gas" or "combined" in ES-CN domain is actually using oil.
                 # Recheck this every 6 months and move to gas key if there has been a change.
                 # Last checked: 2022-06-27
+                MAPPING = PRODUCTION_MAPPING
                 if zone_key.split("-")[1] == "CN":
```

Comment:
I know we talked about using an explicit mapping but since we also assign gas to 0 to be backwards compatible (until I/we update the ree library) we still need the if statement.

Reply:
Ok then let's not over complexify it.